src/DynamoCoreWpf/ViewModels/Core/WorkspaceViewModel.cs

```diff
@@ -358,6 +399,12 @@ void DynamoViewModel_PropertyChanged(object sender, PropertyChangedEventArgs e)
                     // When workspace is changed(e.g. from home to custom), close InCanvasSearch.
                     OnRequestShowInCanvasSearch(ShowHideFlags.Hide);
                     break;
+                case "NodeRunChecked":
```

Comment:
Right. That is the intention here. But today it is different. Freeze can be made on only one node at a time. But that will change when we have the right design for bunch of nodes.